app/lib/common/widgets/user_builder.dart

```diff
@@ -49,6 +61,7 @@ class UserBuilder extends ConsumerWidget {
     final room = ref.watch(briefRoomItemWithMembershipProvider(roomId));
     final invited =
         ref.watch(roomInvitedMembersProvider(roomId)).valueOrNull ?? [];
+    final joined = ref.watch(membersIdsProvider(roomId)).valueOrNull ?? [];
```

Comment:
would also avoid doing that twice if we moved it into the `UserStateButton`...

Reply:
fixed by 08838aa